integration-tests/db/db.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs')
+const db = require('../../server/database')
+
+const deleteRows = (table) => db(table).del()
+
+const clearNotificationTables = async () => {
+  await deleteRows('ShiftNotification')
+  await deleteRows('ShiftTaskNotification')
+}
+
+const destroy = async (done) => {
+  db.destroy(done)
```

Comment:
Does this need an await?

Reply:
Ignore that comment. It just looked out of place without everything else having await.